lm75bd/lm75bd.h

```diff
@@ -5,7 +5,14 @@
 #include <stdint.h>
 
 /* LM75BD I2C Device Address */
-#define LM75BD_OBC_I2C_ADDR /* Define the address here */
+#define LM75BD_OBC_I2C_ADDR 0b01001111 /* Define the address here */
+
+/* Address of Temp Register */
+#define TEMP_REGISTER 0x00
+
+/* R/W Buffers */
+#define WRITE_BUFFER 1
+#define READ_BUFFER 2
```

Comment:
define these in the .c file since they don't need to be accessibly by other files